packages-ts/gauntlet-terra-contracts/src/commands/tooling/updateContractAdmin.ts

```diff
@@ -0,0 +1,80 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgUpdateContractAdmin } from '@terra-money/terra.js'
+import { CATEGORIES } from '../../lib/constants'
+
+type CommandInput = {
+  newAdmin: string
+  contract: string
+  forceNewAdmin?: boolean
+}
+
+export default class UpdateContractAdmin extends TerraCommand {
+  input: CommandInput
+
+  static description = 'Updates contract admin'
+  static examples = [
+    `yarn gauntlet tooling:transfer_contract_admin --network=bombay-testnet --to=<NEW_ADMIN> <CONTRACT_ADDRESS>`,
+  ]
+
+  static id = 'tooling:transfer_contract_admin'
+  static category = CATEGORIES.TOOLING
+
+  constructor(flags, args: string[]) {
+    super(flags, args)
+  }
+
+  buildCommand = async (flags, args): Promise<TerraCommand> => {
+    this.input = this.makeInput(flags, args)
+    this.validateInput(this.input)
+    return this
+  }
+
+  beforeExecute = async () => {
+    await prompt(`Continue transferring contract ${this.input.contract} to new admin ${this.input.newAdmin}?`)
+  }
+
+  makeInput = (flags, args): CommandInput => {
+    return {
+      newAdmin: flags.to,
+      contract: args[0],
+      forceNewAdmin: !!flags.forceNewAdmin,
+    }
+  }
+
+  validateInput = (input: CommandInput): boolean => {
+    if (input.forceNewAdmin) {
+      logger.warn(`New admin "${input.newAdmin}" might not be a multisig wallet`)
```

Comment:
This message could be improved to be more explicit.

Also, new admin address could be checked if it's a known multisig address even if the `--force` flag is set, and print that info to the user.